extension/js/common/core/mime.ts

```diff
@@ -208,7 +210,11 @@ export class Mime {
       } else {
         contentNode = new MimeBuilder('multipart/alternative'); // tslint:disable-line:no-unsafe-any
         for (const type of Object.keys(body)) {
-          contentNode.appendChild(Mime.newContentNode(MimeBuilder, type, body[type]!.toString())); // already present, that's why part of for loop
+          let content = body[type]!.toString();
+          if (type === 'text/html') {
+            content = Mime.extractInlineImagesToAttachments(content, rootNode);
+          }
+          contentNode.appendChild(Mime.newContentNode(MimeBuilder, type, content)); // already present, that's why part of for loop
```

Comment:
`// already present, that's why part of for loop` this comment belongs to lone 213. I know it's not obvious at all - it was referring to the `!` bang, trying to justify it

Reply:
Moved, thanks for catching it 👍